### ISSUE TITLE
[Feature Request] Toggle Display Lock option in context menu.

### ISSUE THREAD

**Issue by gaveitatry2** [Stateford/Display-Lock#121]
Sometimes I want to toggle the Display Lock off to free up my cursor and not have to tab out to other windows. But instead of having to exit the program until I am ready to toggle Display Lock on again and then go looking for the program shortcut to start it up again, it would be nice if we could go to the context menu and select "Toggle Display Lock" to toggle it off/on. Even better if we could add a custom hotkey like "Ctrl Shift D" or "Ctrl Shift L" to toggle it off and on with a keyboard key combo.

**Comment by Stateford** [comment]
![image](https://user-images.githubusercontent.com/15315637/227077401-57594822-c8f2-47e8-a013-001fc704bf63.png)

Currently there is an options to stop it, but not toggle it.

#42 There is an open issue for this, but I haven't gotten around to adding it. The last time I looked at it proved to be more complicated than I anticipated. I will take another look at it, however I'm not quite sure how long it might take.

**Comment by gaveitatry2** [comment]
The Stop is always grayed out. If I open my game that I whitelisted, I can't click on the programs context menu unless I tab out. If I tab out and then go to the context menu, it is grayed out.

I don't want to stop it while it is in use. I want to toggle it off before I even start the game. Then if I want to use it again, I'll just close the game, toggle it back on, and then start the game again. But I can't do that. Since there is no toggle, I have to exit and start the program when needed. I just wanted to clarify that in case there was a misunderstanding.

**Comment by Stateford** [comment]
The stop is meant for the regular use case of display lock. I could add the enable button to the context menu instead.

**Comment by gaveitatry2** [comment]
Yeah, that sounds a lot better. "Enable/Disable" instead of "Toggle Display Lock".

**Comment by gaveitatry2** [comment]
If you do add an Enable/Disable option to the context menu, would you be able to add a global customizable hotkey with two modifiers? Example: "Ctrl Shift D" or "Ctrl Shift L"? I know it's a lot of trouble, but I just think it would be a cool, useful feature. You could just do a one modifier hotkey too, but I have a lot of shortcuts so sometimes being able to use a second modifier key is handy.

**Comment by gaveitatry2** [comment]
@Stateford - I am trying to disable Display Lock temporarily without having to exit the program, but 'Stop' is grayed out and there is no other option to enable or disable Display Lock in the context menu.

**Comment by gaveitatry2** [comment]
I guess this program is abandoned. I created these Issues 11 months ago and there was talk about editing or improving features, but then nothing ever happened.
